smarts/core/chassis.py

```diff
@@ -631,6 +639,10 @@ def control(self, throttle=0, brake=0, steering=0):
         self._apply_throttle(throttle_list)
         self._apply_brake(brake)
 
+    def reapply_last_control(self):
+        assert self._last_control
+        self.control(*self._last_control)
```

Comment:
I remember the cost of a `pybullet` call can be fairly high it looks like there is no way around it.

Reply:
Yeah, but I haven't been able to think of another way around this yet unfortunately.  I also think once everything is functional and stable, we should probably do another profiling pass.